views/default/page/elements/sidebar.php

```diff
@@ -5,6 +5,8 @@
  * @uses $vars['sidebar'] Optional content that is displayed at the bottom of sidebar
  */
 
+echo elgg_view('search/search_box');
```

Comment:
don't know if this will eventually be the best place... but probably better to work with than extending views